pom.xml

```diff
@@ -45,7 +45,7 @@
     <properties>
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
-        <bamboo.version>7.2.1</bamboo.version>
+        <bamboo.version>8.0.4</bamboo.version>
         <bamboo.data.version>7.2.1</bamboo.data.version>
```

Comment:
The project `README` indicates that both `bamboo.version` and `bamboo.data.version` should be increased...?

> Edit the pom.xml file and set <bamboo.version>6.2.1</bamboo.version> and > <bamboo.data.version>6.2.1</bamboo.data.version> (the actual versions will be different to these examples) to the latest Bamboo version. This in turn will mean atlas-run will load the specified Bamboo version.

Reply:
yeah probably should ... but somehow ... still works!